### PR TITLE
[Commands] Add entity variable command

### PR DESCRIPTION
**Pull request by Kinglykrab**
# Commands
- Add `#entityvariable clear` to clear all entity variables from yourself or your target.
- Add `#entityvariable delete [Variable Name]` to delete an entity variable from yourself or your target.
- Add `#entityvariable set [Variable Name] [Variable Value]` to set an entity variable for yourself or your target.
- Add `#entityvariable view [Search Criteria]` to view your or your target's entity variables.

# Notes
- `#entityvariable set` can use multi-word names/values by using double quotes like `#entityvariable set "Test Variable" "Test Value"`.
- `#entityvariable view` does not require a search criteria, not using one shows all entity variables on yourself or your target.

### CODE REVIEW

**Comment by Akkadius** [comment]
Later today turned into next week. Been unreal busy lately

**Comment by Akkadius** [comment]
Can we consolidate this into one command with sub-commands? I feel this goes backwards in your previous efforts of consolidation and creates multiple commands to manipulate the same primitive. 

`commands.cpp` is already a monster to compile and it's less commands to remember

```
#entityvariable [set|delete|clear|list]
```

**Comment by Kinglykrab** [comment]
> Can we consolidate this into one command with sub-commands? I feel this goes backwards in your previous efforts of consolidation and creates multiple commands to manipulate the same primitive.
> 
> `commands.cpp` is already a monster to compile and it's less commands to remember
> 
> ```
> #entityvariable [set|delete|clear|list]
> ```
> 
> Alias (#ev)

Works for me. I’ll consolidate this afternoon.

File: zone/command.cpp [hunk 1]
```diff
@@ -133,6 +131,7 @@ int command_init(void)
 		command_add("emptyinventory", "Clears your or your target's entire inventory (Equipment, General, Bank, and Shared Bank)", AccountStatus::GMImpossible, command_emptyinventory) ||
 		command_add("enablerecipe", "[Recipe ID] - Enables a Recipe", AccountStatus::QuestTroupe, command_enablerecipe) ||
 		command_add("endurance", "Restores your or your target's endurance.", AccountStatus::Guide, command_endurance) ||
+		command_add("entityvariable", "[Clear|Delete|Set|View] - Modify entity variables for yourself or your target", AccountStatus::GMAdmin, command_entityvariable) ||
```
Comment (Akkadius) [inline zone/command.cpp:134]: These should probably be lowercase 

`[clear|delete|set|view]`

**Review by Akkadius — APPROVED** [review]
Minor comments, LGTM

File: zone/command.cpp (hunk 1 above)
Comment (Kinglykrab) [reply zone/command.cpp]: Sounds good.

**Pull request merged**